src/Diagnostics.CompilerHost/appsettings.json

```diff
@@ -18,5 +18,11 @@
     "AADAuthority": "",
     "ClientId": "", // Azure AAd App id of CompilerHost
     "AllowedAppIds": "" // Azure AAD App id of RuntimeHost
+  },
+  "SecuritySettings": {
+    "AllowedCertIssuers": "",
+    "AllowedCertSubjectNames": "",
+    "ShowIdentityModelErrors": false,
+    "AdditionalAllowedCertSubjectNames": ""
```

Comment:
I don't think we'd need `AdditionalAllowedCertSubjectNames` at this point. Hawk created that for Runtime host for some other reason. Given that CompilerHost will be majorily called by Runtimehost, we can remove this.